backend/src/akvo/lumen/lib/import/csv.clj

```diff
@@ -62,6 +62,9 @@
         column-titles
         column-types))
 
+(defn valid-column-name? [column-name]
+  (boolean (re-find #"^c\d+$" column-name)))
```

Comment:
> ;; The distinction is that re-find tries to find _any part_ of the string
> ;; that matches the pattern, but re-matches only matches if the _entire_
> ;; string matches the pattern.

https://clojuredocs.org/clojure.core/re-matches

Reply:
no idea about it :) I've used same approach that we have in akvo.lumen.lib.transformation.engine https://github.com/akvo/akvo-lumen/blob/dataset-by-group-api/backend/src/akvo/lumen/lib/transformation/engine.clj#L112

Reply:
the current code works because we're limiting the behaviour via the regex itself `#"^c\d+$"`, it must start with `c` and then one or more digits, it most end with a digit

Reply:
changed!